addons/mail/static/src/core/common/chat_window_model.js

```diff
@@ -53,11 +53,7 @@ export class ChatWindow extends Record {
         await this._onClose(options);
         this.delete();
         if (escape && indexAsOpened !== -1 && chatHub.actuallyOpened.length > 0) {
```

Comment:
unrelated to diff, but it's strange to name this param `escape`, from the point of view of this function it should be `focusNext` or something like that

Reply:
This variable comes from `options.escape`, which gives context from the close.
We could indeed rename to `focusNext`.